gossip/src/cluster_info.rs

```diff
@@ -671,6 +671,17 @@ impl ClusterInfo {
         self.push_self();
     }
 
+    pub fn set_tpu(&self, socket: SocketAddr) {
+        let _ = self.my_contact_info.write().unwrap().set_tpu(socket);
+
+        self.insert_self();
+        self.push_message(CrdsValue::new_signed(
+            CrdsData::Version(Version::new(self.id())),
+            &self.keypair(),
+        ));
```

Comment:
Is this extra message needed? It seems like you should just need an `insert_self()` and `push_self()` to get the contact info out.

@behzadnouri can you confirm if this message is needed to propagate a change to just the tpu host address?

Reply:
Yes, you are right, my bad, careless copypasta